packages/create-discord-bot/src/helpers/packageManager.ts

```diff
@@ -47,19 +47,52 @@ export function resolvePackageManager(): PackageManager {
  * @param packageManager - The package manager to use
  */
 export function install(packageManager: PackageManager) {
-	let installCommand;
+	let installCommand: string[] | string = `${packageManager} install`;
+
+	console.log(`Installing dependencies with ${packageManager}...`);
 
 	switch (packageManager) {
-		case 'npm':
+		case 'yarn':
+			console.log();
+			installCommand = [
+				`${packageManager} set version stable`,
+				`${packageManager} config set nodeLinker node-modules`,
+				`${packageManager} config set logFilters --json '[{ "code": "YN0002", "level": "discard" }, { "code": "YN0013", "level": "discard" }, { "code": "YN0032", "level": "discard" }, { "code": "YN0060", "level": "discard" }]'`,
+				`${packageManager} plugin import interactive-tools`,
+				`${packageManager} plugin import workspace-tools`,
+				installCommand,
+			];
+			break;
+		case 'deno':
+			installCommand = `${packageManager} cache --reload src/index.ts`;
+			break;
 		case 'pnpm':
-			installCommand = `${packageManager} install`;
+			console.log();
 			break;
-		case 'yarn':
-			installCommand = packageManager;
+		case 'bun':
+			console.log();
```

Comment:
Do we really need console.log in every case?

Reply:
It's only for some to create proper spacing in the terminal output.

Reply:
One console.log before switch statement isnt enough? Without duplicating it in every case